src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,62 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage{
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy (css = ".music .songs")
+            WebElement songsTab;
+    @FindBy (css = "#songsWrapper tr.song-item")
+            WebElement select;
+    @FindBy (css = "[data-testid='play-btn']")
+            WebElement play;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```

For example

![POM_vs_Factory_1](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/3180b7f4-09f1-4c35-9857-4039e5f32d27)

![POM_vs_Factory_2](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/d6bdea75-3b34-495b-a097-be8531f37c30)